__tests__/components/buttons/StyledSwitch.test.tsx

```diff
@@ -145,7 +149,11 @@ describe('<StyledSwitch/>', () => {
         }}
       />,
     );
+    expect(screen.getByTestId('track')).toHaveStyle({
+      backgroundColor: 'blue',
+    });
     fireEvent.press(screen.getByTestId('styled-switch'));
+    act(() => travel(500));
```

Comment:
Because of the animation being fired when the switch is clicked, we had the `Warning: An update to ForwardRef inside a test was not wrapped in act(...).` error in the tests.
I followed this guide to implement time travelling in the tests to mock the animation: https://caian.dev/posts/test-animations-react-native/